lib/produce/itunes_connect.rb

```diff
@@ -92,19 +107,30 @@ def initial_create
     end
 
     def initial_pricing
+      # It's not important here which is the price set. It will be updated by deliver
       sleep 3
       click_on "Pricing"
-      first('#pricingPopup > option[value="3"]').select_option
+      first('#pricingPopup > option[value="0"]').select_option
       first('.saveChangesActionButton').click
     end
 
     private
+      def bundle_exist?
+        if @wildcard_bundle
+          return false # We should always create the app from scratch.
```

Comment:
The problem is where the script checks for the existance of the app.
In the "Bundle ID" dropdown menu of the "New iOS App" you will always have the wildcard bundle id and never the full bundle id.

That part can be improved by checking if the app exist from another place.